guake/terminal.py

```diff
@@ -211,6 +211,15 @@ def configure_terminal(self):
         if hasattr(self, "set_alternate_screen_scroll"):
             self.set_alternate_screen_scroll(True)
 
+        enable_sixel = client.get_boolean("use-sixel")
```

Comment:
Does this need to be a preferences option? Seems like the kind of thing that anyone who has a terminal that can support it would want. If we do keep this as an option in preferences, we also need to add some switches on setting toggle, because as-written currently if a user clicks the box to enable they still won't see sixels until they restart guake.

Reply:
I figured it's probably a good idea to have this optional at least as long as sixel support is still considered experimental in vte.

Reply:
While it is experimental, it's also quite unavailable, I think the only way currently for people to have a version of general VTE installed is to compile it yourself, which is tantamount to opting in, and when it lands in an official release pushed to general users I would think it is no longer experimental.

Reply:
That makes sense. I'll remove the option to enable them and just turn them on when available.

Reply:
I'd probably still keep the setting in the gschema, if advanced users want to still opt-out somehow / for some reason. But I'll default it to enabled, would that be okay?